hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/ReconServerConfigKeys.java

```diff
@@ -60,31 +60,31 @@
 
   public static final String RECON_STORAGE_DIR = "recon";
 
-  public static final String RECON_OM_SOCKET_TIMEOUT =
-      "recon.om.socket.timeout";
-  public static final String RECON_OM_SOCKET_TIMEOUT_DEFAULT = "5s";
+  public static final String OZONE_RECON_OM_SOCKET_TIMEOUT =
+      "ozone.recon.om.socket.timeout";
+  public static final String OZONE_RECON_OM_SOCKET_TIMEOUT_DEFAULT = "5s";
 
-  public static final String RECON_OM_CONNECTION_TIMEOUT =
-      "recon.om.connection.timeout";
-  public static final String RECON_OM_CONNECTION_TIMEOUT_DEFAULT = "5s";
+  public static final String OZONE_RECON_OM_CONNECTION_TIMEOUT =
+      "ozone.recon.om.connection.timeout";
+  public static final String OZONE_RECON_OM_CONNECTION_TIMEOUT_DEFAULT = "5s";
 
-  public static final String RECON_OM_CONNECTION_REQUEST_TIMEOUT =
-      "recon.om.connection.request.timeout";
+  public static final String OZONE_RECON_OM_CONNECTION_REQUEST_TIMEOUT =
+      "ozone.recon.om.connection.request.timeout";
 
-  public static final String RECON_OM_CONNECTION_REQUEST_TIMEOUT_DEFAULT = "5s";
+  public static final String OZONE_RECON_OM_CONNECTION_REQUEST_TIMEOUT_DEFAULT = "5s";
 
-  public static final String RECON_OM_SNAPSHOT_TASK_INITIAL_DELAY =
-      "recon.om.snapshot.task.initial.delay";
+  public static final String OZONE_RECON_OM_SNAPSHOT_TASK_INITIAL_DELAY =
+      "ozone.recon.om.snapshot.task.initial.delay";
```

Comment:
```suggestion
  public static final String OZONE_RECON_OM_SNAPSHOT_TASK_INITIAL_DELAY =
      "ozone.recon.om.snapshot.task.initial.delay";
  @Deprecated
  public static final String RECON_OM_SNAPSHOT_TASK_INITIAL_DELAY = 
      "recon.om.snapshot.task.initial.delay";
```

Reply:
You are right, I just added these deprecated configurations in a new commit!